### ISSUE TITLE
fix: docs about hugo extended version on macOS

### ISSUE THREAD

**Issue by peaceiris** [gohugoio/hugoDocs#823]
@yarko @bep 

About #817 
I use Hugo via `brew` on macOS. This Hugo seems to be an extended version by default.

```
$ hugo version
Hugo Static Site Generator v0.55.6/extended darwin/amd64 BuildDate: unknown
```

[homebrew-core/hugo.rb at master · Homebrew/homebrew-core](https://github.com/Homebrew/homebrew-core/blob/master/Formula/hugo.rb#L22)

_Originally posted by @peaceiris in https://github.com/gohugoio/hugoDocs/pull/817#issuecomment-493723731_

**Comment by bep** [comment]
Is this a question?

**Comment by peaceiris** [comment]
Yes.
I think that we should fix a part of #817 

About this file.
https://github.com/gohugoio/hugoDocs/blob/master/content/en/troubleshooting/faq.md

> The extended is **not** what you get by default, as an example, when you run `brew install hugo` on macOS.

We should fix the above like the following because an extended is default on macOS with `brew install hugo`.

```markdown
The extended is not what you get by default. Only, when you installed Hugo with `brew install hugo` on macOS, the extended is a default.
```

**Comment by korczas** [comment]
just installed hugo with brew, and it's not extended version. 

> go build -ldflags=-s -w -tags extended

It looks like a problem with build itself.
<img width="893" alt="image" src="https://user-images.githubusercontent.com/25669684/185795028-c556a8ee-eac2-45cf-9f14-57445d2d0864.png">
